helpers.sh

```diff
@@ -442,6 +441,18 @@ export_coveralls_data() {
   local service_pull_request=""
   local url="unknown"
 
+  # Change directory to the target project repository.
+  # The assumption here is that the STON file resides within the
+  # repository.
+  # Otherise, don't change directory and presume that we're where
+  # we need to be.
+  if is_file "${config_ston}"; then
```

Comment:
I wonder: should the current working directory take precedence over all of this? We could do it the other way around: if we are in a git repo, we stay there. otherwise, we try to switch to the directory of the ston. if that does not work, we skip this.

Reply:
Sounds reasonable. I'll change that.